mesa/examples/advanced/epstein_civil_violence/app.py

```diff
@@ -36,6 +36,13 @@ def citizen_cop_portrayal(agent):
     return portrayal
 
 
+def post_process(ax):
+    ax.set_aspect("equal")
+    ax.set_xticks([])
+    ax.set_yticks([])
+    ax.get_figure().set_size_inches(10, 10)
```

Comment:
I really like this possibility!

Reply:
yes it gives a lot more control over visual styling